lib/kapost_deploy/task.rb

```diff
@@ -87,21 +87,28 @@ def define
 
     private
 
-    def initialize(name)
+    def initialize(name, scope: Rake.application.current_scope.path)
       defaults
-      self.name = name
+      @name = name
+      @scope = scope
     end
 
     attr_accessor :plugins
+    attr_reader :scope
+
+    def hook_name(type)
+      return "#{name}:#{type}_#{name}" if scope.empty?
+      "#{scope}:#{name}:#{type}_#{name}"
```

Comment:
Nah, lets not add another dependency. This is fine.

We could also have done `.delete_if { |el| el.blank? }`